services/cloudrun-service/Earthfile

```diff
@@ -0,0 +1,79 @@
+VERSION --use-function-keyword 0.7
+IMPORT ../../infrastructure/earthly/go AS go-build
+
+LOCALLY
+ARG --global service=$(basename $PWD)
+ARG --global src_files=$(find pkg -type f ! -name "*_test.go")
+ARG --global cgo_enabled=0
```

Comment:
I would prefer if this was hardcoded, so nobody accidentally adds a cgo library.

Reply:
It cannot be hard-coded. All earthfiles use a common [Earthfile](https://github.com/freiheit-com/kuberpult/blob/main/infrastructure/earthly/go/Earthfile) that is configurable. So the use of a global variable here is the only way to disable cgo.